src/components/calculator.js

```diff
@@ -1,108 +1,105 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import Blocks from './Blocks';
 import calculate from '../logic/calculate';
 
-export default class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-  }
+const Calculator = () => {
+  const [prevState, setState] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
 
-  handleCalculate = (label) => {
-    const { total, next, operation } = this.state;
+  const handleCalculate = (label) => {
+    const { total, next, operation } = prevState;
     const newObj = calculate({ total, next, operation }, label);
-    this.setState({
+    setState({
       total: newObj.total,
       operation: newObj.operation,
       next: newObj.next,
     });
   };
```

Comment:
- [ ] The basic calculator operators (**ex: '+', '-', '/')** of your calculator app are not displayed on the display bar when a user enters it. As the previous reviewer suggested, kindly work on this.